<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>consulter_historique_mach</class>
 <widget class="QDialog" name="consulter_historique_mach">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/sig/erp.jpg);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>750</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="iconnes.qrc">:/sig/images/erp.jpg</pixmap>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>641</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>421</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>chef service maintenance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panne_machines_PB">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>490</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>consulter les pannes de machines</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="type__LE">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>600</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>560</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>consulter les pannes de machines</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>600</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>type machine</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>1351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="planif_ent_pb">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>590</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>planifier un entretien</string>
   </property>
  </widget>
  <widget class="QPushButton" name="recherche_panne_pb">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>600</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>recherche</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>540</y>
     <width>311</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="menu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>190</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>consulter historique des machines</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="iconnes.qrc"/>
 </resources>
 <connections/>
</ui>
